occurrences/event_api_services.py

```diff
@@ -16,6 +16,9 @@
 
 
 def fetch_place_as_json(place_id: str, **filter_params):
+    if not place_id:
+        return None
```

Comment:
The places were fetched without ids https://sentry.hel.ninja/aok/palvelutarjotin-api/issues/11202/